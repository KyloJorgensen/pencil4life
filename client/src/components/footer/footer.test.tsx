'use strict';

import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import Footer from './footer';

let wrapper: ShallowWrapper;

describe('Render Footer', () => {
	it('matches the snapshot', () => {
		wrapper = shallow(
			<Footer>
			</Footer>
		);
		expect(wrapper).toMatchSnapshot();
	});
});