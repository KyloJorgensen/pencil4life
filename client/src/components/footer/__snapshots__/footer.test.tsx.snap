// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EventListenerConsumer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": [Function],
    },
    "ref": null,
    "rendered": [Function],
    "type": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": [Circular],
      "Provider": Object {
        "$$typeof": Symbol(react.provider),
        "_context": [Circular],
      },
      "_calculateChangedBits": null,
      "_changedBits": 0,
      "_changedBits2": 0,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {
        "addEventListener": [Function],
        "removeEventListener": [Function],
      },
      "_currentValue2": Object {
        "addEventListener": [Function],
        "removeEventListener": [Function],
      },
      "_defaultValue": Object {
        "addEventListener": [Function],
        "removeEventListener": [Function],
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": [Function],
      },
      "ref": null,
      "rendered": [Function],
      "type": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": Object {
          "$$typeof": Symbol(react.provider),
          "_context": [Circular],
        },
        "_calculateChangedBits": null,
        "_changedBits": 0,
        "_changedBits2": 0,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": Object {
          "addEventListener": [Function],
          "removeEventListener": [Function],
        },
        "_currentValue2": Object {
          "addEventListener": [Function],
          "removeEventListener": [Function],
        },
        "_defaultValue": Object {
          "addEventListener": [Function],
          "removeEventListener": [Function],
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
